activemq-unit-tests/src/test/java/org/apache/activemq/ProducerFlowControlTest.java

```diff
@@ -261,7 +261,7 @@ public void testDisableWarning() throws Exception {
 
         Appender appender = new DefaultTestAppender() {
```

Comment:
Instead of the DefaultTestAppender, you can now use log4j AbstractAppender with var as I did in other modules.